pkg/shim/scheduler.go

```diff
@@ -256,7 +265,9 @@ func (ss *KubernetesShim) canHandle(se events.SchedulerEvent) bool {
 func (ss *KubernetesShim) schedule() {
 	apps := ss.context.SelectApplications(nil)
 	for _, app := range apps {
-		app.Schedule()
+		if app.Schedule() {
+			ss.outstandingAppsFound = true
+		}
```

Comment:
We should be able to simplify this to:
```
ss.outstandingAppsFound = app.Schedule()
```

Reply:
We cannot simplify it because if in a cycle we found some outstanding apps, but in the next one not, we don't want to owerride it.

Reply:
We cannot simplify it because if in a cycle we found some outstanding apps, but in the next one not, we don't want to owerride it.

Reply:
ok